harness/determined/common/api/request.py

```diff
@@ -356,3 +363,37 @@ def ws(host: str, path: str) -> WebSocket:
     token = authentication.must_cli_auth().get_session_token()
     websocket.add_header("Authorization".encode(), "Bearer {}".format(token).encode())
     return WebSocket(websocket)
+
+
+def parse_warnings(
+    warnings: Optional[Union[Sequence[bindings.v1LaunchWarning], Sequence[int]]]
+) -> Optional[Sequence[bindings.v1LaunchWarning]]:
+
+    # Warnings may be passed in as either an enum value or integer value
+    warning_message_int_to_enum_value = {
+        1: bindings.v1LaunchWarning.LAUNCH_WARNING_CURRENT_SLOTS_EXCEEDED
+    }
```

Comment:
When would this happen?

Reply:
removed this. Relates to: https://github.com/determined-ai/determined/pull/5376#discussion_r1028420540